test/base/src/androidTest/kotlin/io/realm/kotlin/test/shared/RealmSetTests.kt

```diff
@@ -58,7 +59,10 @@ class RealmSetTests {
     private lateinit var realm: Realm
 
     private val managedTesters: List<SetApiTester<*, RealmSetContainer>> by lazy {
-        descriptors.map {
+        descriptors.filter {
+            // Filter out MutableRealmInts
+            it.elementType.classifier != MutableRealmInt::class
```

Comment:
Why?

Reply:
As I mentioned we don't support this data type for lists, collections or sets in Java so I went for the same constraint.